packages/ra-core/src/controller/field/useReferenceManyFieldController.ts

```diff
@@ -178,6 +181,7 @@ export const useReferenceManyFieldController = <
             pagination: { page, perPage },
             sort,
             filter: filterValues,
+            meta,
         },
```

Comment:
You should also pass the remaining quetry options to the query, see e.g. https://github.com/marmelab/react-admin/blob/5e2ed7d3f43c0a849ae33e18737ef83429dad639/packages/ra-core/src/controller/edit/useEditController.ts#L99